### DISCUSSION TITLE
Web Project - Duvida

### DISCUSSION THREAD

**Discussion by EriksonLoures** [V1ctorW1ll1an/WebProject#25] [Q&A] [answered]
Qual arquitetura você esta usando no projeto?

**Comment by v1ctorw1ll1an** [comment]
Estou utilizando algo parecido com clean arc

**Comment by v1ctorw1ll1an** [comment, marked as answer]
Da uma olhada neste blog https://blog.cleancoder.com/uncle-bob/2012/08/13/the-clean-architecture.html
